gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -225,6 +225,15 @@ private void showProjectSelector() {
     ProjectSelectorDialogFragment.show(getFragmentManager());
   }
 
+  private void showBasemapSelector() {
+    viewModel.showBasemapSelector();
+  }
```

Comment:
Since this is only accessible from the OfflineAreaManagerFragment maybe it should be moved there? Also please add TODO to rename it to something more consistent (OfflineAreaSelector?).

Reply:
done